tools/depends/target/libdcadec/Makefile

```diff
@@ -0,0 +1,38 @@
+include ../../Makefile.include
+DEPS= ../../Makefile.include Makefile
+
+# lib name, version
+LIBNAME=dcadec
+ARCHIVE=master.tar.gz
+BASE_URL=https://github.com/foo86/dcadec/archive
```

Comment:
please upload a tagged version to our mirrors instead. Let me know if I should do that for you.